src/base/utils/fs.cpp

```diff
@@ -287,6 +288,99 @@ QString Utils::Fs::tempPath()
     return path;
 }
 
+namespace
+{
+#ifndef Q_OS_WIN
+/*
+ * Borrowed from a stackexchange answer
```

Comment:
@zeule:  Please provide an actual link to the originating answer:  

[<https://stackoverflow.com/a/758054>]

Reply:
Ah, thanks! I have found it myself.

Reply:
Pardon, have *not* found it myself.